docs/system_overview/data_objects.md

```diff
@@ -0,0 +1,343 @@
+ - `resc_group_name`: This column is not touched by iRODS anymore. It refers to the legacy resource groups feature. This should contain the value `EMPTY_RESC_GROUP_NAME` in order to discourage legacy queries.
+ - `resc_name`: This column is not touched by iRODS anymore. This is the name of the storage resource which hosts this replica. This should contain the value `EMPTY_RESC_NAME` in order to discourage legacy queries.
+ - `data_path`: Physical path of the data in storage for this replica. This is a string which is determined by the storage resource plugin hosting the data. For example, the unixfilesystem resource will specify a slash-delimited fully qualified path to a file on disk in the vault.
+ - `data_owner_name`: The username of the owner of the data object that maps to this replica. This column will contain the same value for all replicas of a given data object. This value is independent from the Access Control List for the data object which allows for multiple owners to exist. The value of this column will never change, even if the user listed is removed.
+ - `data_owner_zone`: The zone of the user that owns the data object that maps to this replica. This column will contain the same value for all replicas of a given data object. This will be populated even if the owner is a user in the local zone.
+ - `data_is_dirty`: This is the status of the replica which is represented as an integer from 0 to 4, inclusive. See [Replica Status](#replica-status) for more details about these statuses.
+ - `data_status`: This column is not touched by iRODS anymore. Its purpose is lost to history.
+ - `data_checksum`: String representing the checksum of the replica. This may vary between replicas depending on the state of the data and may or may not reflect the state of the data in storage. `ifsck -K` can be used to detect inconsistencies.
+ - `data_expiry_ts`: This column is not touched by iRODS anymore. Its purpose is lost to history.
+ - `data_map_id`: This column is not touched by iRODS anymore. Its purpose is lost to history.
+ - `data_mode`: **Help! The purpose of this column is unknown, but it is still used by iRODS.**
+ - `r_comment`: This column is not touched by iRODS anymore. Its purpose is lost to history.
+ - `create_ts`: Timestamp which indicates the time that this replica was created, represented as seconds since the epoch. This value will never be changed after the replica is created.
+ - `modify_ts`: Timestamp which indicates the most recent time that this replica was modified, represented as seconds since the epoch. This will be updated every time this replica is opened for write and modified.
+ - `resc_hier`: This column is not touched by iRODS anymore. This is the semicolon-delimited resource hierarchy which ends with the storage resource which hosts this replica. This should contain the value `EMPTY_RESC_HIER` in order to discourage legacy queries.
```

Comment:
"This column is no longer used by iRODS.  The value `EMPTY_RESC_HIER` is used to discourage legacy queries."

Reply:
Done